[2024-06-19T09:50:00.297+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T09:50:00.313+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [queued]>
[2024-06-19T09:50:00.321+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [queued]>
[2024-06-19T09:50:00.321+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T09:50:00.336+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:10:00+00:00
[2024-06-19T09:50:00.338+0530] {standard_task_runner.py:63} INFO - Started process 2565 to run task
[2024-06-19T09:50:00.340+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:10:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpp2dhy7kj']
[2024-06-19T09:50:00.342+0530] {standard_task_runner.py:91} INFO - Job 88: Subtask backup_task
[2024-06-19T09:50:00.388+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [running]> on host FN-21052948.
[2024-06-19T09:50:00.460+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:10:00+00:00'
[2024-06-19T09:50:00.461+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T09:50:00.469+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T09:50:00.470+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T09:50:00.474+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T09:50:00.482+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:50:00.488+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:50:00.502+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:50:00.507+0530] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T09:50:00.712+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T09:50:00.713+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T09:50:00.713+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T09:50:00.714+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T09:50:00.750+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:10:00+00:00, execution_date=20240619T041000, start_date=20240619T042000, end_date=20240619T042000
[2024-06-19T09:50:00.794+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T09:50:00.841+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T09:50:00.850+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-19T09:53:05.844+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T09:53:05.863+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [queued]>
[2024-06-19T09:53:05.870+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [queued]>
[2024-06-19T09:53:05.870+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T09:53:05.891+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:10:00+00:00
[2024-06-19T09:53:05.894+0530] {standard_task_runner.py:63} INFO - Started process 2669 to run task
[2024-06-19T09:53:05.897+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:10:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpmdx4jbd6']
[2024-06-19T09:53:05.898+0530] {standard_task_runner.py:91} INFO - Job 92: Subtask backup_task
[2024-06-19T09:53:05.953+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:10:00+00:00 [running]> on host FN-21052948.
[2024-06-19T09:53:06.052+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:10:00+00:00'
[2024-06-19T09:53:06.052+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T09:53:06.065+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T09:53:06.066+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T09:53:06.071+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T09:53:06.080+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:53:06.086+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:53:06.102+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T09:53:06.108+0530] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T09:53:06.351+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T09:53:06.352+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T09:53:06.352+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T09:53:06.353+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T09:53:06.388+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:10:00+00:00, execution_date=20240619T041000, start_date=20240619T042305, end_date=20240619T042306
[2024-06-19T09:53:06.431+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T09:53:06.458+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T09:53:06.466+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
